crates/collab/src/llm/authorization.rs

```diff
@@ -6,21 +6,45 @@ use crate::{Config, Error, Result};
 
 pub fn authorize_access_to_language_model(
     config: &Config,
-    _claims: &LlmTokenClaims,
+    claims: &LlmTokenClaims,
     country_code: Option<String>,
     provider: LanguageModelProvider,
     model: &str,
 ) -> Result<()> {
-    authorize_access_for_country(config, country_code, provider, model)?;
-
+    authorize_access_for_country(config, country_code, provider)?;
+    authorize_access_to_model(config, claims, provider, model)?;
     Ok(())
 }
 
+fn authorize_access_to_model(
+    config: &Config,
+    claims: &LlmTokenClaims,
+    provider: LanguageModelProvider,
+    model: &str,
+) -> Result<()> {
+    if config.is_development() {
```

Comment:
We don't need this here, do we?

I would think we'd want development to operate the same as production.